package.json

```diff
@@ -12,7 +12,7 @@
     ]
   },
   "scripts": {
-    "start": "NODE_CONFIG_DIR=./server/config node server",
+    "start": "NODE_CONFIG_DIR=./server/config babel-node server",
```

Comment:
Will fix this in next PR https://github.com/OpenCollective/opencollective-website/pull/427